crates/key-server/src/server.rs

```diff
@@ -502,6 +519,7 @@ async fn handle_get_service(
     Ok(Json(GetServiceResponse {
         service_id: app_state.server.key_server_object_id,
         pop: app_state.server.key_server_object_id_sig,
+        version: PACKAGE_VERSION.to_string(),
```

Comment:
we're not using response header for this?

Reply:
Added in https://github.com/MystenLabs/seal/pull/84/commits/9f91cad169a406a60c471ca670a11ce5cd4826e4. I think it makes sense to keep it here also though, to make it easy to query.